Makefile.org

```diff
@@ -418,12 +418,22 @@ clean:	libclean
 	rm -f $(LIBS)
 	rm -f openssl.pc libssl.pc libcrypto.pc
 	rm -f speed.* .pure
+	$(RM) test/newkey.pem test/testkey.pem test/testreq.pem
+	-$(RM) `find . -name '*.o' -a \! -path "./.git/*"`
```

Comment:
But this is supposed to be taken care of in recursive clean. Which are not removed?